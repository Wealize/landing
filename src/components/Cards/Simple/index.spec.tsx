import React from 'react'
import { mount } from 'enzyme'

import SimpleCard from './index'

describe('<SimpleCard />', () => {
  const cardProps = {
    children:
      'We are a digital product studio with the upper hand in blockchain, conversational agents, cognitive services, mobile apps and web development'
  }
  const wrapper = mount(<SimpleCard {...cardProps} />)

  it('wrapper: should render', () => {
    expect(wrapper).toBeDefined()
    expect(wrapper).toMatchSnapshot()
  })

  it('should render h3 text content correctly', () => {
    expect(wrapper.find('h3').text()).toEqual(cardProps.children)
  })
})
