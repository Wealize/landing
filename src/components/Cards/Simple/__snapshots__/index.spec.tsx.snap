// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SimpleCard /> wrapper: should render 1`] = `
<Memo(SimpleCard)>
  <styles__Container
    className="simple-card"
  >
    <div
      className="styles__Container-sc-58pc36-0 cvYCcG simple-card"
    >
      <styles__Text>
        <h3
          className="styles__Text-sc-58pc36-1 euGlbe"
        >
          We are a digital product studio with the upper hand in blockchain, conversational agents, cognitives services, mobile apps and web development
        </h3>
      </styles__Text>
    </div>
  </styles__Container>
</Memo(SimpleCard)>
`;
